app/controllers/couches/photos_controller.rb

```diff
@@ -0,0 +1,20 @@
+class Couches::PhotosController < ApplicationController
+  def new
+    @photo = CouchPhoto.new
+  end
+
+  def create
+
+    couch = Couch.find(params["couch_id"])
+    @photo = couch.photos.new(photo_params)
+    if @photo.save!
+      redirect_to user_couch_path(current_user, couch)
+    end
```

Comment:
else flash[:danger] & redirect_to new_couch_path? though we can easily throw this in down the road